### ISSUE TITLE
Derived ID proofing and AAL

### ISSUE THREAD

**Issue by LarryFrank** [usnistgov/800-63-3#597]
### All Fields Are Required
**Organization Name (N/A, if individual):**
N/A 

**Organization Type (see below for codes)**:
4
 
**Document (63-3, 63A, 63B, or 63C)**:
63A

**Reference (Include section and paragraph number)**:
6.2, 6.3
 
**Comment (Include rationale for comment)**:
 "derivation" of identity should not change the AAL level (what the current text seems to say) it may change the IAL level inherited from the original token.  The strength of the authentication mechanism is not impacted by the identity proofing method.  The strength of the ID proofing maybe impacted by the AAL of the original authenticator.  A hardware based PKI cryptographic module is not changed by not collecting a biometric at ID proofing.  But an IAL3 ID proofing may be impacted by use of an AAL2 authenticator. 
**Suggested Change**:
Modify 6.2 and 6.3 to specify what is required to claim IAL2 or IAL3 when using a derive identity.  E.G., If the original ID proofing is IAL3 and presented via an AAL3 technical mechanism, it is IAL2 UNLESS the id proofing is done in person and collects a biometric.  If the AAL2 of the original token is 2, then the IAL is 2.
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by paul-grassi** [comment]
AAL being completed separated from IAL allows CSPs to issue any AAL authenticator and bind it to the identity IAL of the original proofing.  So we aren't going to create requirements that state If IAL2 then you can only give an AAL3 authenticator.

**Comment by LarryFrank** [comment]
Completely missed the point of the comment - in fact got it backwards.  Section 6.2.and 6.2 of 63A changes the AAL level based on the type of ID proofing.  Agree that IAL: and AAL OUGHT to be completely separate - but these two sections create a linkage between the process of ID proofing and the AAL level of the resulting credential.

**Comment by paul-grassi** [comment]
we will reevaluate.

**Comment by LarryFrank** [comment]
I think the confusion was my attempt to suggest appropriate language.  I think the intent of the current language is to say that a "remote" ID proofing is less trustworthy than in person, and that is especially when the original IAL is 3 (which required in person proofing).  But modifying the AAL seems the wrong way to approach it.   I was suggesting that a better way would be to specify how the derived IAL would be impacted by different scenarios.

**Comment by DoD-CIO-ID** [comment]
Discussed with Larry Frank, who approved the following language to further explain his initial comment: 

"Regarding derived credentials, it's important to distinguish between the IAL, which relates to the issuance process of the credential, and the AAL, which relates to the security of the credential and authenticators themselves.  A credential derived from an IAL-3 credential may itself be IAL-3, but this has no bearing on its AAL.  

Section 6.2 and 6.3 should clarify that the AAL of the primary authenticator has no bearing on the AAL of the derived credential.  Instead these sections should be about derived credential requirements for IAL-2 and IAL-3, respectively.

Lastly, sections 6.2 and 6.3 should clarify that checking the status of the primary authenticator means checking whether the primary authenticator has expired or been revoked."

**Comment by jimfenton** [comment]
This is a bit confusing because authenticator requirements (otherwise covered in -63B) are mixed in with derived credentials here. We will be moving the authenticator requirements to -B and give the CSP the option to issue authenticators that are subsidiary (dependent) on others, in the same way that derived PIV is subsidiary to PIV.

**Comment by paul-grassi** [comment]
We decided to remove derived credentials altogether for 2 reasons, which will be placed in the document to smooth this transition.

1.  Derived PIV need not be 'abstracted' in 800-63.  800-157 is the auth source for that and should remain.
2.  Identity evidence for proofing purposes, already can be physical or digital.  In the digital case, you prove the evidence is yours by proving possession of an authenticator. The proof could be a signed assertion from an IDP.  

So complicating this document to retain a tie to PIV is overkill and obviously confusing. And we want to blur physical and digital evidence, so we decided on this path.
